tests/unit/test_bruteforce.cpp

```diff
@@ -371,6 +371,34 @@ TEST_F(BruteForceTest, brute_force_vector_search_by_id_test) {
     VecSimIndex_Free(index);
 }
 
+TEST_F(BruteForceTest, search_for_nothing) {
```

Comment:
Consider adding this check to an exiting basic test such as `brute_force_vector_search_test_l2` maybe, to reduce the inflation of the test files...